en/flight_modes/return.md

```diff
@@ -15,26 +15,52 @@ The specific behaviour for each vehicle type is described below.
+
+![Return mode cone](../../assets/flying/rtl_cone.jpg)
+
+If the vehicles is:
+- Above [RTL_RETURN_ALT](#RTL_RETURN_ALT) (1) it will return at its current altitude.
+- Below the cone it will return where it intersects the cone (2) or [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) (whichever is higher).
+- Outside the cone (3) it will first climb until it reaches [RTL_RETURN_ALT](#RTL_RETURN_ALT).
+- Inside the cone:
+  - Above [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) it will return at its current altitude.
+  - Below [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) (4) it will first ascend to `RTL_DESCEND_ALT`.
+
+Note:
+- If [RTL_CONE_ANG](#RTL_CONE_ANG) is 0 degrees there is no "cone":
+  - the vehicle returns at `RTL_RETURN_ALT` (or above).
+  - This is the default behaviour.
+  - This is the only behaviour prior to PX4 v1.9.
+- If [RTL_CONE_ANG](#RTL_CONE_ANG) is 90 degrees the vehicle will return at the greater of `RTL_DESCEND_ALT` and the current altitude.
+- The vehicle will always ascend at least [RTL_DESCEND_ALT](#RTL_DESCEND_ALT) for the return.
 
-> **Note** The `RTL_LAND_DELAY` is provided to allow time for landing gear to be deployed (this is triggered automatically). By default this period is short so that the vehicle will simply slow and then land immediately. The parameter can also be set so that the vehicle will hover indefinitely.
 
 The RTL behaviour can be configured using the parameters below.
 
 
 Parameter | Description
 --- | ---
-<span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Minimum RTL return altitude in meters (default: 60m). The vehicle will ascend to this altitude before returning. If already above this value the vehicle will return at its current altitude.
-<span id="RTL_DESCEND_ALT"></span>[RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | Altitude at which the vehicle will slow or stop its initial descent (default: 30m)
-<span id="RTL_LAND_DELAY"></span>[RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | Time to hover at `RTL_DESCEND_ALT` before landing (default: 0.5s). If set to -1 the system will loiter at `RTL_DESCEND_ALT` rather than landing.
-<span id="RTL_MIN_DIST"></span>[RTL_MIN_DIST](../advanced_config/parameter_reference.md#RTL_MIN_DIST) | Minimum horizontal distance from home position to trigger ascent to a safe altitude (RTL_RETURN_ALT). If the vehicle is horizontally closer than this distance to home, it will return at its current altitude (instead of first ascending to RTL_RETURN_ALT). 
+<span id="RTL_RETURN_ALT"></span>[RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Return altitude in meters (default: 60m) when [RTL_CONE_ANG](../advanced_config/parameter_reference.md#RTL_CONE_ANG) is 0. If already above this value the vehicle will return at its current altitude.

```

Comment:
@RomanBapst You need to check the firmware parameter definition on all these - may no longer be correct.

Reply:
@hamishwillee Will do!